lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2719,18 +2720,24 @@ def wait_for_customization(self, vm, poll=10000, sleep=10):
                     eventsFinishedResult = self.get_vm_events(vm, ['CustomizationSucceeded', 'CustomizationFailed'])
                     if len(eventsFinishedResult):
                         if not isinstance(eventsFinishedResult[0], vim.event.CustomizationSucceeded):
-                            self.module.fail_json(msg='Customization failed with error {0}:\n{1}'.format(
-                                eventsFinishedResult[0]._wsdlName, eventsFinishedResult[0].fullFormattedMessage))
+                            self.module.warn("Customization failed with error {%s}:{%s}"
```

Comment:
Add a note in porting guide so that users will get to know that customization timeout warns rather than failing. And, users can modify their playbooks accordingly.

Reply:
Add this in wait_for_customization parameter description. Thanks.